pkg/browsers/console.go

```diff
@@ -284,9 +290,13 @@ func makeDestinationURL(service string, region string) (string, error) {
 
 	dest := prefix + serv + "/home"
 
-	//NOTE here: excluding iam here and possibly others as the region isnt in the uri of the webpage on the console
-	if region != "" || serv != "iam" {
+	//excluding region here if the service is apart of the global service list
+	//uncomplete list of global services
+	_, notGlobal := globalServiceMap[service]
```

Comment:
The map lookup will return true if the item is in the map, so _, global := globalServiceMap["iam"], global would be true

```suggestion
	_, global := globalServiceMap[service]
```

Reply:
Good catch, ill fix it up